ibm/data_source_ibm_cis_waf_groups_test.go

```diff
@@ -0,0 +1,33 @@
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccIBMCisWAFGroupsDataSource_Basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMCisWAFGroupsDataSourceConfigBasic1(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_cis_waf_groups.waf_groups", "waf_groups.0.group_id"),
+					resource.TestCheckResourceAttrSet("data.ibm_cis_waf_groups.waf_groups", "waf_groups.0.modified_rules_count"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMCisWAFGroupsDataSourceConfigBasic1() string {
+	return testAccCheckIBMCisDomainDataSourceConfigBasic1() + fmt.Sprintf(`
+	data "ibm_cis_waf_groups" "waf_groups" {
+		cis_id     = data.ibm_cis.cis.id
+		domain_id  = data.ibm_cis_domain.cis_domain.id
+		package_id = "c504870194831cd12c3fc0284f294abb"
```

Comment:
can we get id from resource instead of hardcoding

Reply:
datasource is not planned for getting the group_id. so hardcoded it